dirsrvtests/tests/suites/webui/ldap_browser/ldap_browser_test.py

```diff
@@ -70,6 +87,220 @@ def test_ldap_browser_tab_visibility(topology_st, page, browser_name):
     assert frame.locator('#searchBase').is_visible()
 
 
+def test_create_and_delete_user(topology_st, page, browser_name):
+    """ Test to create and delete user
+
+    :id: eb08c1d7-cbee-4a37-b724-429e1cdfe092
+    :setup: Standalone instance
+    :steps:
+         1. Call load_LDAP_browser_tab function.
+         2. Click on ou=people.
+         3. Call create_entry function to create new user.
+         4. Check that new user is successfully created.
+         5. Click on newly created user.
+         6. Call delete_entry function to delete ou.
```

Comment:
Just a small nitpick 'Call delete_entry function to delete user' :)
Otherwise LGTM and tests are passing.